lib/app/features/feed/create_post/views/pages/post_form_modal/post_form_modal.dart

```diff
@@ -178,6 +181,13 @@ class PostFormModal extends HookConsumerWidget {
     usePreselectLanguage(ref, eventReference: modifiedEvent);
     useDetectLanguage(ref, enabled: parentEvent == null, quillController: textEditorController);
 
+    useNsfwValidation(
+      ref: ref,
+      mediaFiles: attachedMediaFilesNotifier.value,
+      videoFile: attachedVideoNotifier.value,
+    );
+    useInvalidateOnDispose(ref, mediaNsfwCheckerProvider);
```

Comment:
We should invalidate `mediaNsfwCheckerProvider` manually any time we use `useNsfwValidation`? 
If so, maybe put it inside the `useNsfwValidation`?